kiwi/firmware.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, xml_state):
         ]
 
         firmware_types = Defaults.get_firmware_types()
-        if self.host_architecture.startswith('ppc64'):
+        if not self.firmware and self.host_architecture.startswith('ppc64'):
             self.firmware = 'ofw'
```

Comment:
I was thinking about the initial setup of the firmware on the ppc arch. Actually do you think we really should enforce it ?